### PR TITLE
DBZ-3163 Documentation clarification

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.redhat.com/browse/DBZ-3163

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@roldanbob, could you review this one? Thx!

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 1]
```diff
@@ -23,9 +23,22 @@ The first time a {prodname} Db2 connector connects to a Db2 database, the connec
...
-The connector uses the abstract syntax notation (ASN) libraries that come as a standard part of Db2 LUW (Db2 for Linux, UNIX and Windows) and which you can add to Db2 zOS. To use ASN and hence this connector, you must have a license for the IBM InfoSphere Data Replication (IIDR) product. However, IIDR does not need to be installed.
+[NOTE]
+====
+The connector uses the abstract syntax notation (ASN) libraries that come as a standard part of Db2 LUW (Db2 for Linux, UNIX and Windows) and which you can add to Db2 zOS.
+To use ASN and hence this connector, you must have a license for the IBM InfoSphere Data Replication (IIDR) product.
+However, IIDR does not need to be installed.
+====
+
+ifdef::community[]
+The Db2 connector has been tested with Db2/Linux {linux-version}.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:34]: The Db2 connector has been tested with Db2 on Linux {linux-version}.

File: documentation/modules/ROOT/pages/connectors/db2.adoc [hunk 2]
```diff
@@ -23,9 +23,22 @@ The first time a {prodname} Db2 connector connects to a Db2 database, the connec
...
+It is expected that the connector would also work on other platforms such as Windows,
+and we'd love to get your feedback if you can confirm this to be the case.
+endif::community[]
 
-The Db2 connector has been tested with Db2/Linux {linux-version}. It is expected that the connector would also work on Windows, AIX and zOS.
+ifdef::product[]
+The Db2 connector has been tested with Db2/Linux {linux-version}.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/db2.adoc:40]: The Db2 connector was tested with Db2 on Linux {linux-version}.

**Comment by roldanbob** [comment]
A couple of small recommendations, otherwise, LGTM.

File: documentation/modules/ROOT/pages/connectors/db2.adoc (hunk 1 above)
Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/db2.adoc:34]: That's actually not a version of Linux, but of Db2 (on Linux). I'll rename the variable and merge.

**Comment by gunnarmorling** [comment]
@roldanbob, renamed the variable to "db-version. I reckon you'll want to do the same in downstream docs config?

**Comment by roldanbob** [comment]
> @roldanbob, renamed the variable to "db**2**-version. I reckon you'll want to do the same in downstream docs config?

You had me confused, because I was scanning for a variable with the name `db-version`. I finally figured it out after my local Antora build complained about the missing variable. :)

**Pull request merged**